src/event/trigger.js

```diff
@@ -10,13 +10,16 @@ define( [
 
 "use strict";
 
-var rfocusMorph = /^(?:focusinfocus|focusoutblur)$/;
+var rfocusMorph = /^(?:focusinfocus|focusoutblur)$/,
+	stopPropagationCallback = function( e ) {
```

Comment:
The difference is the following:

* If we define the function within the `trigger` method, it is defined every time trigger is called.
* If we define the function within the AMD callback function as I did here, the function will be defined only once, but even if the `trigger` method is never used. Additionally, there is an additional scope lookup.

All in all, these considerations are micro optimisations, that are not really worth considering. The questions is more one of culture within a project. We could also define it at the end of the file, just before the `return jQuery` statement.

I will look at the other files and see how it is usually handled within the `jQuery` project.